app/views/instructor/courses/show.html.erb

```diff
@@ -6,7 +6,7 @@
           type: 'PUT',
           url: ui.item.data('update-url'),
           dataType: 'json',
-          data: { lesson: { row_order_position: ui.item.index() - 1 } }
+          data: { lesson: { row_order_position: ui.item.index() } }
```

Comment:
The index itself is a zero-based index, which is what `row_order_position` expects.  No need to subtract 1 here.